pkg/pod/pod.go

```diff
@@ -322,6 +327,39 @@ func (b *Builder) Build(ctx context.Context, taskRun *v1beta1.TaskRun, taskSpec
 		return nil, err
 	}
 
+	readonly := true
```

Comment:
curious why do we need this?

Reply:
The ReadOnly in the VolumeSource below requires a pointer to a bool variable. Hence created this here.